src/counts.py

```diff
@@ -7,7 +7,10 @@
 count = {}
 # Count the characters in `x`` and put the counts in `counts`.
 # Your code goes here.
-
+for c in x:
+    if c not in count:
+      count[c] = 0 
+    count[c] += 1
```

Comment:
Looks perfect.

This was exactly the answer I was looking for, but as a bonus I'll let you know that Python already supports this functionality:

```python
>>> from collections import Counter
>>> x = 'foobar'
>>> Counter(x)
Counter({'o': 2, 'f': 1, 'b': 1, 'a': 1, 'r': 1})
```

This is usually the right way to do it, but the `Counter` solution has some drawbacks that sometimes make the dictionary solution superior. Now you know how to use both :)